README.md

```diff
@@ -364,6 +364,24 @@ New characters:
 
 `j`, `<`, `>`, `=`, `"`, `/`
 
+### Calling method with more than one argument
+
+Calling method with more than one argument is non trivial - to do it you can use following [technique](https://stackoverflow.com/q/63601330/860099) (discovered by trincot) - for example  example: `"truefalse".replace("true","1")` can be written as follows:
```

Comment:
"method" should be "a method" or "methods"

Reply:
"you can use following" should be "you can use the following"

Reply:
"for example  example" should be "for example"

Reply:
fixed